Shravdev2021

shravanisamala opened the pull request.

LeahHunt committed:
Finished draft version with 3 state stoplight

LeahHunt committed:
messy but functional; still has a couple of bugs

LeahHunt committed:
Theoretically fully functional version of the app

LeahHunt committed:
Added alt text to plots

LeahHunt committed:
Fixed Error in writeImpl #2 and adjusted some code formatting

LeahHunt committed:
added matrixcalc to list of packages

LeahHunt committed:
Made corrections based on Dr. Pearl's feedback; several wording changes, fixed grading issue, fixed issues with the feedback not clearing

LeahHunt committed:
fixed instruction 3's grammar to make it more clear

shravanisamala committed:
- Added title width
- updated upper buttons
- Added copyright for references pages
- fixed code (added inputId, labels, etc)
- Changed psu logo
- Fixed spacing issues in code
- Is there a way to make the New sample button in the second tab centered? Still trying to figure this out

shravanisamala committed:
- Fixed white space in server and updated Description file

shravanisamala committed:
Description not working

shravanisamala committed:
Fixed desription file

shravanisamala committed:
Commit

shravanisamala committed:
Commit

shravanisamala committed:
Finished Prereq Table

shravanisamala committed:
Changed table widths

shravanisamala committed:
Commit

shravanisamala committed:
Made changes Dr. Pearl requested.

shravanisamala committed:
Final commit

hy2021-IT reviewed (dismissed):
The issue on GitHub has been largely resolved. There is generally no major error made while running the app, except for a minor error occurred as the image of the plot failed to appear in some rare occasion. This can be resolved by restarting Rstudio

neilhatfield commented:
Hey @hy2021-IT, if there are any errors, even minor, those need to be addressed before this can be approved. Do you think that the errors can be fixed in a day or less of work?

neilhatfield committed:
first set of changes

working on DRY-ing the code and making small fixes

neilhatfield committed:
Update app.R

Reactivate two libraries

neilhatfield committed:
Update app.R

Layout improvements and changes to server side for the candy/cookie and weather contexts. Traffic lights still need updating.

neilhatfield committed:
Finalizing Updates

neilhatfield committed:
Small tweaks

neilhatfield committed:
Merge pull request #4 from EducationShinyAppTeam/neilDevelopment

Release modifications
